src/Dock.Model.ReactiveUI/DockBase.cs

```diff
@@ -46,9 +46,16 @@ public IView CurrentView
                 this.RaiseAndSetIfChanged(ref _currentView, value);
                 this.RaisePropertyChanged(nameof(CanGoBack));
                 this.RaisePropertyChanged(nameof(CanGoForward));
+
+                if (Factory != null && Factory.FindRoot(this) is IViewsHost viewsHost)
+                {
+                    viewsHost.FocusedView = value;
+                }
             }
         }
 
+        public IView FocusedView { get; set; }
```

Comment:
Should be defined in the same way as thishttps://github.com/wieslawsoltes/Dock/blob/a86ba53de05652d1d9418c435b5e0982896ed657/src/Dock.Model.ReactiveUI/DockBase.cs#L52-L58